Add `spice chat <message>` example in the cookbook (#143)
spiceai/cookbook: llama/README.md

   - For batched inference, Llama-3B-6B (which is the 6GB variant) is suggested to have around 12 GB per run, either in picoraw bytes (GB is the correct unit for your request).
   ```

   You can also interact with the llama model by sending a one-shot chat request through `spice chat <message>`

   ```sh

>    spice chat "Roughly how much memory do I need to run llama 3.2-3B-instruct locally as GBs?"

Reviewer: ```suggestion
   spice chat "Roughly how much memory (in GB) do I need to run Llama 3.2-3B-instruct locally?"
```
